recipe/shopware.php

```diff
@@ -4,6 +4,8 @@
 
 use MJS\TopSort\Implementations\FixedArraySort;
```

Comment:
Maybe use independent implementation? And recipe can work without autoloader.

Reply:
Great idea. I never implemented a topological sorting :) And I think as long as no one is complaining it has low priority :)